src/components/FeaturedProject.css

```diff
@@ -49,25 +49,29 @@
 }
 
 .project__button {
-  border: 3px solid var(--primary-green);
-  background-color: rgb(43, 48, 53);
+  border: 1px solid var(--primary-green);
+  background-color: var(--secondary-grey);
   border-radius: 30px;
   width: 40%;
   height: 30px;
   padding-top: 7px;
   margin: auto;
   margin-top: 20px;
   transition: 200ms;
+  display: block;
 }
 
 .project__content a {
   color: white;
   text-decoration: none;
 }
+
 .project__button:hover,
 .project__button:focus {
-  background-color: var(--primary-green);
+  background-color: white;
+  color: black;
```

Comment:
Everything looks good! I would also suggest making the button more dynamic on hover by adding a transform and/or box shadow. You can check out our styling for the footer icons for inspo here: https://github.com/Tech-Start-UCalgary/tsu-website/blob/main/src/components/SocialMedia/SocialMedia.css#L13